src/client/components/Navigation/BrokerBalance/BrokerBalance.js

```diff
@@ -0,0 +1,113 @@
+import { sortBy } from 'lodash';
+import React, { useEffect, useState } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { Dropdown, Icon, Menu } from 'antd';
+import {
+  getPlatformNameState,
+  getUserWalletState,
+} from '../../../../investarena/redux/selectors/platformSelectors';
+import CurrencyItem from '../../../wallet/CurrencyItem/CurrencyItem';
+import { getUserStatistics } from '../../../../investarena/redux/actions/platformActions';
+import Loading from '../../Icon/Loading';
+import './BrokerBalance.less';
+
+const BrokerBalance = ({ beaxyBalance, platformName, getStatistics }) => {
+  const [initFirstCurrency, setInitFirstCurrency] = useState({});
+  const [initSecondCurrency, setInitSecondCurrency] = useState({});
+  const storageFirstItem = localStorage.getItem('firstCurrency');
+  const storageSecondCurrency = localStorage.getItem('secondCurrency');
```

Comment:
use 'store' lib to deal with local storage (https://github.com/Waiviogit/waivio/blob/dev-beaxy/src/client/helpers/localStorageHelpers.js)